lib/path-utils.ts

```diff
@@ -14,11 +14,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { normalize, relative } from 'path';
+import { posix } from 'path';
 
-// Export all of node's path functions, so that users of this
+// Export all of node's POSIX path functions, so that users of this
 // module need only import this file
-export * from 'path';
+export const basename = posix.basename;
+export const delimiter = posix.delimiter;
+export const dirname = posix.dirname;
+export const extname = posix.extname;
+export const format = posix.format;
+export const join = posix.join;
+export const isAbsolute = posix.isAbsolute;
+export const normalize = posix.normalize;
+export const parse = posix.parse;
+export const relative = posix.relative;
+export const resolve = posix.resolve;
+export const sep = posix.sep;
```

Comment:
I think you can write this:
```typescript
export const {
    basename,
    delimiter,
    dirname,
    extname,
    format,
    join,
    isAbsolute,
    normalize,
    parse,
    relative,
    resolve,
    sep,
} = posix;
```

Reply:
Yes, it works and it is shorter! Done. 👍